client_test/directdefund_test.go

```diff
@@ -83,27 +77,4 @@ func TestDirectDefund(t *testing.T) {
 		}
 	}
 
-	// test failure of teardown of a ledger channel currently funding a virtual channel
```

Comment:
Did you mean to remove this code? I don't quite see the motivation / connection to not swallowing the error?

Reply:
Yup, although I should have made it clear why. Now that we're no longer swallowing the error, it bubbles up to engine where it [panics](https://github.com/statechannels/go-nitro/blob/f41a8b62e5da8bd5cc11f0e682dcc9149f79a71b/client/engine/engine.go#L151).

Since the engine run loop is in a different go routine, it's pretty hard to catch the panic in the test. So I just removed the test for now.